docs/source/basic_setup.rst

```diff
@@ -0,0 +1,113 @@
+
+To get started we will create an empty file to hold the submission credentials and a directory
+for storing the submission Excel files and associated raw sequencing files. At this time, the sequencing
+files need to be on your local machine to be submitted to CGAP. Do this by using the `touch` and `mkdir`
+commands described above. Use the exact command below for creating the credential file, but you can
+create the directory for the submission files anywhere, just note the location (you can check with `pwd`).
+
+
+.. code-block:: bash
+
+    $ touch ~/.cgap-keys.json
+    $ mkdir Documents/submit_cgap
+
+
+Installing Python and Pyenv
+---------------------------
+
+Most systems come with versions of Python installed by default, but oftentimes they are not the
+newest versions, and for our software we prefer to be running newer supported versions of Python
+for security reasons. We also recommend using `pyenv` for managing virtual environments. This allows
+you to isolate Python package installations from one another, so you do not install another package
+with conflicting dependencies that may causes issues.
+
+Begin by installing pyenv using the automatic installer.
+
+.. code-block:: bash
+
+    curl -L https://raw.githubusercontent.com/pyenv/pyenv-installer/master/bin/pyenv-installer | bash
+
+Then add the following to your `~/.bashrc` file. It may have no contents - if it does not exist you can
+copy the below as is and drop it into the file. Doing so ensures that you can use your `~/.bashrc` file
+as a macro for making `pyenv` and associated commands available to you easily.
+
+.. code-block:: bash
+
+    export PYENV_ROOT=“$HOME/.pyenv”
+    command -v pyenv >/dev/null || export PATH=“$PYENV_ROOT/bin:$PATH”
+    eval “$(pyenv init -)”
+    eval “$(pyenv virtualenv-init -)”
+
+Once done you can force the changes to take effect by running `source ~/.bashrc`. Once done you should
+be able to run `pyenv`. In order to install newer Python versions, you will need to install Xcode
+Developer Tools.
+
+.. code-block:: bash
+
+    xcode-select --install
+
+This will take some significant time, but when it has completed you will have the toolchain necessary
+for installing newer Python versions to your machine and can proceed with the below.
+
+.. code-block:: bash
+
+    pyenv install 3.8.13
+
+This command will install Python version 3.8.13 through `pyenv`. If it is not successful feel free
+to copy the error output and send it to the CGAP team. Once the installation has completed, we will
+create and activate a virtual environment for using SubmitCGAP.
+
+.. code-block:: bash
+
+    pyenv virtualenv 3.8.13 submit_cgap38
+    pyenv activate submit_cgap38
+
+This creates a virtual environment called `submit_cgap38` using Python version 3.8.13. We add `38` at
+the end to indicate it is a 3.8 environment. If successful, at this point you can transition to the
+installation docs section Installing SubmitCGAP in a Virtual Environment.
```

Comment:
Independent of my other remarks above, I also don't like this business of using digits mushed together as the convention. The dots are there for a reason. It's not a decimal number. 311 might mean 3.11 or 3.1.1 once you remove the dots, for example.